orchagent/srv6orch.cpp

```diff
@@ -365,18 +382,22 @@ void Srv6Orch::doTaskSidTable(const KeyOpFieldsValuesTuple & tuple)
     SWSS_LOG_ENTER();
     string sid_name = kfvKey(tuple);
     string op = kfvOp(tuple);
-    string sid_list;
+    string sid_list, sidlist_type;
 
     for (auto i : kfvFieldsValues(tuple))
     {
         if (fvField(i) == "path")
         {
           sid_list = fvValue(i);
         }
+        if (fvField(i) == "type")
+        {
+          sidlist_type = fvValue(i);
+        }
```

Comment:
could you update test_srv6.py with this new 'path' attribute in the APP_DB.

Reply:
have added test case for sidlist_type.

Reply:
@kperumalbfn please help review, thanks.